smssync/src/main/java/org/addhen/smssync/presentation/task/state/State.java

```diff
@@ -52,7 +52,7 @@ public boolean isInitialState() {
     }
 
     public boolean isRunning() {
-        return EnumSet.of(SyncState.SYNC).contains(state);
+        return state == SyncState.SYNC;
```

Comment:
Certainly is null-safe, but I was only using it because using `==` on enums is safe, and quicker to type ;¬)  Can change to `.equals()` if you prefer.